io.edgehog.devicemanager.CellularConnectionStatus.json

```diff
@@ -0,0 +1,41 @@
+{
+  "interface_name": "io.edgehog.devicemanager.CellularConnectionStatus",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "mappings": [
+    {
+      "endpoint": "/%{id}/carrier",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Connectivity carrier operator name"
+    },
+    {
+      "endpoint": "/%{id}/registrationStatus",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "GSM/LTE registration status. Possible valueas: [NotRegistered, Registered, SearchingOperator, RegistrationDenied, Unknown, RegistereRoaming]"
+    },
+    {
+      "endpoint": "/%{id}/rssi",
+      "type": "double",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Signal strenght of the device in dBm"
+    },
+    {
+      "endpoint": "/%{id}/technology",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Access Technology. Possible values [Gsm, GsmCompact, Utran, GsmEgprs, UtranHsdpa, UtranHsupa, UtranHsdpaHsupa, Eutran]"
```

Comment:
I would capitalize acronyms, but I'm not sure. CC @rbino

Reply:
Initially, I did, but I had trouble when two terms were near each other and both were acronyms like `UTRAN w/ HSUPA`.